libminifi/src/core/ProcessGroup.cpp

```diff
@@ -485,4 +487,15 @@ ParameterContext* ProcessGroup::getParameterContext() const {
   return parameter_context_;
 }
 
+std::string ProcessGroup::buildGroupPath() const {
+  std::lock_guard<std::recursive_mutex> lock(mutex_);
+  std::string path = name_;
+  auto parent = parent_process_group_;
+  while (parent != nullptr) {
+    path.insert(0, parent->getName() + " / ");
```

Comment:
Do we really want to put spaces around the '/'?  There aren't any in the path example in BulletinStoreTests.cpp.

Reply:
I wanted to keep the format used in NiFi, and additional spaces are used there: https://github.com/apache/nifi/blob/344799defc9b2820f97c12776024d49f4d98ca4d/nifi-framework-bundle/nifi-framework/nifi-framework-core-api/src/main/java/org/apache/nifi/events/BulletinFactory.java#L60

Reply:
OK; should there be spaces at line 41 in BulletinStoreTests.cpp, too?

Reply:
Good point, I updated this and also the assertions in the tests in https://github.com/apache/nifi-minifi-cpp/pull/1920/commits/c6e1ac79dcfb180e540793ff910295aeaa3363cd